packages/cojson/src/sync.ts

```diff
@@ -729,10 +686,8 @@ export class SyncManager {
       if (peer.erroredCoValues.has(coValue.id)) continue;
 
       if (peer.optimisticKnownStates.has(coValue.id)) {
-        await this.tellUntoldKnownStateIncludingDependencies(coValue.id, peer);
         await this.sendNewContentIncludingDependencies(coValue.id, peer);
       } else if (peer.isServerOrStoragePeer()) {
-        await this.subscribeToIncludingDependencies(coValue.id, peer);
         await this.sendNewContentIncludingDependencies(coValue.id, peer);
```

Comment:
Before https://github.com/garden-co/jazz/pull/1741/files we needed to send a `load` message to the server to trigger the load of the CoValue from storage before sending content.

Now it's not required anymore.